docs/docs/api/Authentication/signup.md

```diff
@@ -16,72 +16,72 @@ color: '#E83E8C',
     "Email": string
```

Comment:
For consistency, `Username`, `Password` and `Email` should be lowercased. (I.e. Since you lowercased the example request below, this template should also be lowercased as well)